molecule_ec2/cookiecutter/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml

```diff
@@ -41,10 +24,10 @@ collections:
     - name: Create security groups
       ec2_group:
         vpc_id: "{{ item.subnets[0].vpc_id }}"
-        name: "{{ security_group_name }}"
-        description: "{{ security_group_name }}"
-        rules: "{{ security_group_rules }}"
-        rules_egress: "{{ security_group_rules_egress }}"
+        name: "{{ molecule_yml.driver.security_group_name }}"
```

Comment:
I would have expected to see defaults here, I do not want to prevent users from using default module behaviors: https://docs.ansible.com/ansible/latest/collections/amazon/aws/ec2_group_module.html

Reply:
@ssbarnea  you mean putt here al parameter of the ec2_group_module here in order to permit to manage
it from driver and if not present use default ? If yes I'll do it